tests/test_hrex_rbfe.py

```diff
@@ -129,15 +129,43 @@ def sample_and_record_rss(*args, **kwargs):
     # Initial permutation should be the identity
     np.testing.assert_array_equal(result.hrex_diagnostics.replica_idx_by_state_by_iter[0], np.arange(n_windows))
 
+    # Check that we can extract replica trajectories
+    n_atoms = result.final_result.initial_states[0].x0.shape[0]
+    rng = np.random.default_rng(2024)
+    n_atoms_subset = rng.choice(n_atoms)
```

Comment:
It looks like `rng.choice` can return 0, which I believe would be invalid here

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1320/commits/696d57efe2376bbcefa5ba346c3af451e1432828